lib/unified_health_data/adapters/renewability_helper.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module UnifiedHealthData
```

Comment:
This code is tested as part of other code, but this has no unit test of its own.